lib/time.py

```diff
@@ -23,3 +62,67 @@ def sleep(secs):
 
 def time():
   return float(Now().UnixNano()) / Second
+
+
+_strftime_directive_map = {
+    '%': '%',
+    '0': NotImplementedError,
+    '1': NotImplementedError,
+    '2': NotImplementedError,
+    '3': NotImplementedError,
+    '4': NotImplementedError,
+    '5': NotImplementedError,
+    '6': NotImplementedError,
+    '7': NotImplementedError,
+    '8': NotImplementedError,
+    '9': NotImplementedError,
+    'a': 'Mon',
+    'A': 'Monday',
+    'b': 'Jan',
+    'B': 'January',
+    'c': NotImplementedError,
+    'd': '02',
+    'H': '15',
+    'I': '03',
+    'j': NotImplementedError,
+    'L': '.000',
+    'm': '01',
+    'M': '04',
+    'p': 'PM',
+    'S': '05',
+    'U': NotImplementedError,
+    'W': NotImplementedError,
+    'w': NotImplementedError,
+    'X': NotImplementedError,
+    'x': NotImplementedError,
+    'y': '06',
+    'Y': '2006',
+    'Z': 'MST',
+    'z': '-0700',
+}
+
+def strftime(format, tt=None):  # pylint: disable=missing-docstring,redefined-builtin
```

Comment:
Two blank lines before module-level functions and classes please. Probably makes sense to move the directive map to the top of the file so that the module functions are all grouped together.